src/result/sfProvarAutomationTestRunResult.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  ErrorHandler,
+  GenericErrorHandler,
+  SfProvarCommandResult,
+  SFProvarResult,
+} from '@provartesting/provardx-plugins-utils';
+import { Messages } from '@salesforce/core';
+
+/* eslint-disable */
+export class SfProvarAutomationTestRun extends SFProvarResult {
+  /**
+   * Declaring return type and populating return object for async run method of the commands.
+   *
+   */
+
+  public populateResult(
```

Comment:
Why do we need a separate implementation for populateResult here?

Reply:
need to log separator and output file msg for this command.
Line no: 25